arrows/kpf/yaml/kpf_exception.h

```diff
@@ -48,7 +48,7 @@ namespace vital {
 // ------------------------------------------------------------------
 /// Generic kpf exception
 class  KPF_YAML_EXPORT kpf_exception
-  : public vital_core_base_exception
+  : public vital_exception
```

Comment:
Since this is already in the vital namespace, I wonder if the name is perhaps a little redundant? I might have gone with `base_exception` or even just `exception`. (Not terribly important, though.)